openedx/core/djangoapps/schedules/management/commands/send_recurring_nudge.py

```diff
@@ -4,11 +4,5 @@
 
 class Command(SendEmailBaseCommand):
     async_send_task = ScheduleRecurringNudge
-
-    def __init__(self, *args, **kwargs):
-        super(Command, self).__init__(*args, **kwargs)
-        self.log_prefix = 'Scheduled Nudge'
-
-    def send_emails(self, *args, **kwargs):
-        for day_offset in (-3, -10):
-            self.enqueue(day_offset, *args, **kwargs)
+    log_prefix = 'Scheduled Nudge'
```

Comment:
Since these are class-specific values, we chose to define them as class variables.  Passing them through the constructor would make them instance values, which don't need to be different for each run of the test.  Also, it might be a stylistic thing - but making them class variables make them stand out as declarative and away from the application code.